packages/components/Item/components/ItemHeader.js

```diff
@@ -65,6 +65,14 @@ const ActionButton = styled(Button)`
   margin-left: 16px;
 `;
 
+const PathHintWrapper = styled.div`
+  margin-right: auto;
+`;
+
+const HintWrapper = styled.div`
```

Comment:
Use styledHint instead of addon redundant wrapper. The same concern for PathHintWrapper